verification/certifyKey.go

```diff
@@ -118,6 +36,14 @@ func TestCertifyKeySimulation(d TestDPEInstance, c DPEClient, t *testing.T) {
 	testCertifyKey(d, c, t, true)
 }
 
+func TestDiceTcbInfo(d TestDPEInstance, c DPEClient, t *testing.T) {
+	testDiceTcbInfo(d, c, t, false)
+}
+
+func TestDiceTcbInfoSimulation(d TestDPEInstance, c DPEClient, t *testing.T) {
+	testDiceTcbInfo(d, c, t, true)
+}
+
```

Comment:
Made as a separate test case because, TestDiceTcbInfo requires derived context for various fields validation. Deriving child context requires to use MakeDefault, ChangeLocality(for Sim mode) when used as a part of "AllTestCases". So made it as separate testcase.